src/modules/journal/journal.selector.ts

```diff
@@ -41,17 +41,10 @@ export const hasSlotsAfterSelectedDate = (journal: JournalModel): boolean => {
 };
 
 export const canNavigateToNextDay = (journal: JournalModel): boolean => {
-  let traverseWeekend = false;
   const nextDayAsDate = DateTime.at(journal.selectedDate).add(1, Duration.DAY).format('YYYY-MM-DD');
   const dayAfterTomorrowAsDate = DateTime.at(DateTime.today()).add(2, Duration.DAY).format('YYYY-MM-DD');
-  const nextDayAsDay = DateTime.at(journal.selectedDate).add(1, Duration.DAY).day();
 
-  // if the current day is a Friday(5) or Saturday(6), allow navigation to the Monday.
-  if ((DateTime.at(DateTime.today()).day() === 5 || DateTime.at(DateTime.today()).day() === 6) && nextDayAsDay !== 2) {
-    traverseWeekend = true;
-  }
-
-  return (nextDayAsDate < dayAfterTomorrowAsDate) || traverseWeekend || hasSlotsAfterSelectedDate(journal);
+  return (nextDayAsDate < dayAfterTomorrowAsDate) || hasSlotsAfterSelectedDate(journal);
```

Comment:
Should it only be the `hasSlotsAfterSelectedDate`. Can really think of a scenario when we would navigate to the next day and there are no slots on that day.

Reply:
I thought the user would need to be able to **navigate** to the next day even if there are no slots. Otherwise how would they verify that they have no slots?

Reply:
Uhh you are right. A weekend with no slots would be the best example.. Not sure how I missed that

Reply:
So am i right in saying that the first check is correct then?